src/libcore/iter.rs

```diff
@@ -8,51 +8,278 @@
 // option. This file may not be copied, modified, or distributed
 // except according to those terms.
 
-//! Composable external iterators
+//! Composable external iteratoration
 //!
-//! # The `Iterator` trait
+//! If you've found yourself with a collection of some kind, and needed to perform an operation on
```

Comment:
I'm hesitant to use that word since "streaming iterators" are something we don't support, and I'm worried about confusion. But you're right. I'm not sure if this is broad enough.